redbadger/website-honestly
# Create page for bank case study

### Motivation

A new case study about our work at a one of the work's largest banks.

To view, manually go to the page `/our-work/case-study/financial-services-digital-transformation`.

### Things to note

This also fixes the style of quotes for all recent case studies (Sky, Camden Market, ...). The CSS clashed with the style we have for older case studies like [BMW](https://red-badger.com/our-work/case-study/bmw/) or [Haller](https://red-badger.com/our-work/case-study/haller/).

| Before | After |
| --- | --- |
| ![Camden Market Quote Before](https://user-images.githubusercontent.com/3579251/34486931-020a08de-efca-11e7-8e15-4a9207fa36e0.png) | ![Camden Market Quote After](https://user-images.githubusercontent.com/3579251/34486939-05ab8e68-efca-11e7-8dc2-86c49958ecac.png) |

### Test plan

- Open the bank case study in every browser and make sure it looks and behaves the same.
- Open other case studies and check if the quote looks fine (should have red quotation marks now).

### Pre-merge checklist
